trevhud/inHouse_webapp
# Refactor of python scripts for pi

To test:

1. Ensure you are connected to the same router as the pi.
2. SSH into the pi via: `ssh pi@raspberrypi.local`
3. Move to inHouseProduce directory `cd inHouseProduce`
4. remove configuration files `rm out.txt ihp_config.ini`
5. run capture script `python3 multi_capture_A.py&`
6. Kill script after ~15 seconds

- [ ] Ensure that there are no errors while running
- [ ] Ensure that `ihp_config.ini` and `out.txt` are created
- [ ] Ensure that a file is sent to s3 (the location will be provided by the script output)
